using-mobile-extensions/adobe-journey-optimizer/README.md

```diff
@@ -2,38 +2,38 @@
 
 The [Adobe Journey Optimizer](https://business.adobe.com/products/journey-optimizer/adobe-journey-optimizer.html) extension for Adobe Experience Platform Mobile SDKs powers push notifications for your mobile apps. This extension helps you collects user push tokens and manages interaction measurement with Adobe Experience Platform services.
 
-The following documentation details how to use the extension as well as the required configuration across Adobe Experience Platform services, app stores, and your apps to get started with push notifications for Adobe Journey Optimizer.
+The following documentation details how to use the extension and required configurations to get started with push notifications for Adobe Journey Optimizer for Adobe Experience Platform services, app stores, and your apps.
 
 ## Before starting
 
-1. Follow the steps in [this page](https://experienceleague.adobe.com/docs/journey-optimizer/using/get-started/configuration/push-config/push-gs.html?lang=en) to configure push channel in Adobe Journey Optimizer.
-2. [Update your app's Datastream](./#update-datastream-with-profile-dataset) in [Adobe Experience Platform Data Collection](https://launch.adobe.com)
+1. Read the tutorial on [getting started with push configuration](https://experienceleague.adobe.com/docs/journey-optimizer/using/get-started/configuration/push-config/push-gs.html?lang=en) to learn how to configure push channels in Adobe Journey Optimizer.
+2. [Update your app's Datastream](#update-datastream-with-profile-dataset) in [Adobe Experience Platform Data Collection](https://experience.adobe.com/#/data-collection/)
 3. Integrate with following extensions:
    * [Mobile Core](../../foundation-extensions/mobile-core/)
    * [Adobe Experience Platform Edge Network](../../foundation-extensions/experience-platform-extension/)
    * [Identity for Edge Network](../../foundation-extensions/identity-for-edge-network/)
 
 ### Update Datastream with Profile Dataset
```

Comment:
H3 - should this be :
 “### Update Datastream with a Profile Dataset”
 or is it branded like RTCDP?